setup.py

```diff
@@ -113,12 +113,12 @@ def get_extras_require() -> Dict[str, List[str]]:
             "pytorch-lightning>=1.0.2,<1.5.0",
             "skorch",
             "catalyst>=21.3",
-            "torch==1.8.0 ; sys_platform=='darwin'",
-            "torch==1.8.0+cpu ; sys_platform!='darwin'",
-            "torchvision==0.9.0 ; sys_platform=='darwin'",
-            "torchvision==0.9.0+cpu ; sys_platform!='darwin'",
-            "torchaudio==0.8.0",
-            "allennlp>=2.2.0,<2.7.0",
+            "torch==1.10.0 ; sys_platform=='darwin'",
+            "torch==1.10.0+cpu ; sys_platform!='darwin'",
+            "torchvision==0.11.1 ; sys_platform=='darwin'",
+            "torchvision==0.11.1+cpu ; sys_platform!='darwin'",
+            "torchaudio==0.10.0",
+            "allennlp>=2.2.0 ; python_version>'3.6'",
             "botorch>=0.4.0 ; python_version>'3.6'",
```

Comment:
[Question] Don't we have to update the lower constraint for botorch as well?

Reply:
`pytest tests/integration_tests/test_botorch.py` doesn't show any error with botorch==0.4.0 and botorch==0.5.1.

Reply:
Ah, so sorry, you are totally right. I forgot to delete this comment when submitting the review...

Reply:
No worries. In fact, I did not check them. Fortunately, there was no error :)